src/stories/Other/Search.stories.js

```diff
@@ -14,8 +14,8 @@ const SearchStories = {
 };
 
 const fetchUsers = async (search) => {
-    const results = await fetch("https://dummyapi.io/data/api/user?limit=100", {
-        headers: { "app-id": "lTE5abbDxdjGplutvTuc" }, // Some random dude's API key.
```

Comment:
Doesn't work anymore due to CORS. We need another free API or some other fancy setup